ghcup-vanilla-0.0.9.yaml

```diff
@@ -5970,6 +5970,91 @@ ghcupDownloads:
               dlHash: c31042043f074447a2a1fa4a2d3789c0a63975ee6a253e161b67b4b9ceb5f2f8
           Linux_UnknownLinux:
             unknown_versioning: *cabal-31411-arm64
+    3.14.2.0:
+      viTags:
+        - Latest
```

Comment:
If you add `Latest` to a new version, you have to remove it from the old version.

Reply:
Done. I've almost move the `Recommended` tag.